src/bin/cargo/commands/help.rs

```diff
@@ -15,7 +15,20 @@ const COMPRESSED_MAN: &[u8] = include_bytes!(concat!(env!("OUT_DIR"), "/man.tgz"
 pub fn cli() -> Command {
     subcommand("help")
         .about("Displays help for a cargo subcommand")
-        .arg(Arg::new("COMMAND").action(ArgAction::Set))
+        .arg(Arg::new("COMMAND").action(ArgAction::Set).add(
+            clap_complete::ArgValueCandidates::new(|| {
+                // TODO: Need a faster way to get all subcommand names.
```

Comment:
Unless we have something telling us this is slow, we shouldn't bother with this TODO, especially tracking it in source.